packages/ui/atomic/create-atomic-component-project/template/src/components/results-manager/results-manager.tsx

```diff
@@ -0,0 +1,35 @@
+import {Component, h} from '@stencil/core';
```

Comment:
Should we not include any component and put all the templates directly in the index.html?

Reply:
wdym?

Reply:
results-manager.tsx... should it be part of the starter project?
or if all the templates, should be added directly into index.html?

Reply:
Mhhh I don't know xD
I'll go with the public vote.

Reply:
For a whole search page it was cleaner to have this component and split the templates in different files. Here I figure it's more of a hassle. You could just add it to the HTML

Reply:
I'm fine with either